cmd/di.go

```diff
@@ -416,3 +416,8 @@ func (di *Dependencies) bootstrapLocationComponents(options node.OptionsLocation
 	di.LocationDetector = location.NewDetector(di.IPResolver, di.LocationResolver)
 	di.LocationOriginal = location.NewLocationCache(di.LocationDetector)
 }
+
+var corsConfig = tequilapi.CorsConfig{
+	DefaultTrustedOrigin:  "https://mysterium.network",
+	AllowedOriginSuffixes: []string{"wallet.mysterium.network", "wallet-dev.mysterium.network", "localhost"},
```

Comment:
"localhost" is needed just for `--localnet` case, lets add new CLI argument `--wallet.address`